tensorflow/python/ops/bincount_ops_test.py

```diff
@@ -163,6 +164,50 @@ def test_dense_input(self,
     self.assertAllEqual(expected_values, y.values)
     self.assertAllEqual(expected_shape, y.dense_shape)
 
+  @parameterized.named_parameters(
+    {
+        "testcase_name": "_baseline",
+        "x": np.array([1, 1, 2, 3, 2, 4, 4, 5], dtype=np.int32),
+        "expected_values": [0, 2, 2, 1, 2, 1]
+    }, {
+        "testcase_name": "_no_maxlength",
+        "x": np.array([[3, 2, 1], [5, 4, 4]], dtype=np.int32),
+        "expected_values": [[0, 1, 1, 1, 0, 0],[0, 0, 0, 0, 2, 1]]
+    }, {
+        "testcase_name": "_maxlength",
+        "x": np.array([[3, 2, 1, 7], [7, 0, 4, 4]], dtype=np.int32),
+        "maxlength": 7,
+        "expected_values": [[0, 1, 1, 1, 0, 0, 0],[1, 0, 0, 0, 2, 0, 0]]
+    }, {
+          "testcase_name": "_minlength",
+          "x": np.array([[3, 2, 1, 7], [7, 0, 4, 4]], dtype=np.int32),
+          "minlength": 9,
+          "expected_values": [[0, 1, 1, 1, 0, 0, 0, 1, 0], 
+                              [1, 0, 0, 0, 2, 0, 0, 1, 0]]
+    })
```

Comment:
It seems that in the file we don't have a quite complete named_parameter test with full dense args (input, weights, etc.)

So I've started to duplicate the test related to the `dense_input` where only the input is dense and I've adapted it for the "full dense" op (as we don't support sparsity).

I don't know if it makes sense to fully duplicate/adapt all the `dense_input` named_parameters (here in WIP). 

I'm talking about this because I want to try to keep the test economy a bit safe without adding any other unnecessary test to TF's (already) very long execution list.

Let me know what do you think when you have checked the original test list.